### PR TITLE
LE audio move broadcast sink cbs

### PR DESCRIPTION
**Pull request by Thalley**
Moves the callbacks used for the broadcast sink role.

### CODE REVIEW

**Review by cvinayak — COMMENTED** [review]
Not familiar with functional changes, hence reviewed for style. Old declarations appeared to be reverse Christmas tree, not sure if it needs to be maintained as part of changes in this PR.

**Comment by Thalley** [comment]
> Not familiar with functional changes, hence reviewed for style. Old declarations appeared to be reverse Christmas tree, not sure if it needs to be maintained as part of changes in this PR.

In any case I've updated it to follow the reverse Christmas tree.

**Review by cvinayak — APPROVED** [review]

**Review by asbjornsabo — APPROVED** [review]
LGTM

File: subsys/bluetooth/host/audio/broadcast_sink.c [hunk 1]
```diff
@@ -34,6 +34,8 @@
 static struct bt_audio_broadcast_sink broadcast_sinks[BROADCAST_SNK_CNT];
 static struct bt_le_scan_cb broadcast_scan_cb;
 
+static sys_slist_t sink_cbs = SYS_SLIST_STATIC_INIT(&sink_cbs);
```
[the remark this answers is not in the document]
Comment (Thalley) [reply subsys/bluetooth/host/audio/broadcast_sink.c:37]: @asbjornsabo I see what you mean. The naming was used to be similar to e.g. the `scan_cbs` and `pa_sync_cbs` from scan.c.

**Pull request merged**